### ISSUE TITLE
Marking episodes nondeletable and adding drag'n'drop in playlist; crashing issue

### ISSUE THREAD

**Issue by IstvanKohany** [deltakosh/Podcasts#30]
So, I have two propositions of UI enhancements and one remark:

I wish I could:

1. mark episodes as non-deletable by the "delete downloads older than" option.

2. have the ability to drag and drop episodes in playlist. Right now I have to annoyingly choose the move up/down from the context menu to sort episodes in order I'd like.

And remark: after rosolving the #28 (thanks again for that!) the mobile app crashes more frequently. I can't reproduce them in any logical manner: sometimes its during updates, sometimes during removing episodes from playlist, sometimes in options menu :-)

**Comment by deltakosh** [comment]
Good ideas
For the crash do you reproduce on PC? Else it is mostly due to phone OS instability when using insiders build

**Comment by IstvanKohany** [comment]
> For the crash do you reproduce on PC?

No, the PC version is stable. You think that turning insider builds off could help?

**Comment by deltakosh** [comment]
Yep
